wp-parsely.php

```diff
@@ -84,6 +82,19 @@ function maybe_load_plugin() {
 
 		require $entry_file;
 
+		return;
+	}
+}
+add_action( 'muplugins_loaded', __NAMESPACE__ . '\maybe_load_plugin' );
+
+function maybe_disable_some_features() {
+	if ( ! apply_filters( 'wpvip_parsely_load_mu', get_option( '_wpvip_parsely_mu' ) === '1' ) ) {
```

Comment:
Seems like this line should be extracted into a function that returns a boolean, since its the same condition as in the function above.